main.rkt

```diff
@@ -22,6 +25,9 @@
 
 ;; =============================================================================
 
+(define (square n)
+  (expt n 2))
+
```

Comment:
I think we should use Racket's `factorial`. (I'd be happy if some day, lots of these functions from `scmutils` get absorbed into the Racket math library.)
http://docs.racket-lang.org/math/number-theory.html?q=fact#%28def._%28%28lib._math%2Fnumber-theory..rkt%29._factorial%29%29